Cargo.lock

```diff
@@ -650,9 +650,9 @@ dependencies = [
 
 [[package]]
 name = "cipherstash-client"
-version = "0.22.0"
+version = "0.22.1"
```

Comment:
Nit: I think we should be explict with the version in Cargo.toml

Reply:
Not exactly sure what you mean here. It's set to this version in Cargo.toml, unless you mean something more than that